internal/services/apimanagement/api_management_resource.go

```diff
@@ -729,13 +729,11 @@ func resourceApiManagementServiceCreateUpdate(d *pluginsdk.ResourceData, meta in
 				},
 			}
 
-			future, err := client.CreateOrUpdate(ctx, id.ResourceGroup, id.ServiceName, params)
+			// We only handle the error here because no request body is returned https://github.com/Azure/azure-rest-api-specs/issues/23456
+			_, err := client.CreateOrUpdate(ctx, id.ResourceGroup, id.ServiceName, params)
```

Comment:
To give a little context here: this is returning an LRO, so there won't be a HTTP Response **by design** - this API behaviour is intentional and not an API bug. If you wish to query the state of a resource that's being provisioned as an LRO, you'll need to either poll on the LRO (using `WaitForCompletion` in the Track1 SDK) - or by polling on the resource and checking the `provisioningState` (as below) - although the LRO shouldn't be returning until this is completed, therefore polling on the LRO **should** be sufficient here